tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/LedgerCommandGroup.java

```diff
@@ -42,6 +43,7 @@
         .withCategory(CATEGORY_LEDGER_SERVICE)
         .addCommand(new SimpleTestCommand())
         .addCommand(new DeleteLedgerCommand())
+        .addCommand(new UpdateLedgersCommand())
```

Comment:
I think this command is converting  the bookie id between ip address and hostname. it is required to run on a specific bookie. I would suggest adding this command to `BookieCommandGroup` and call this command as `flip-bookie-id` or `change-bookie-id`.

Reply:
ok, I will fix it later